app/models/person.rb

```diff
@@ -0,0 +1,18 @@
+class Person < ActiveRecord::Base
+	has_many :tasks
+
+	def completed_tasks
+		tasks = self.tasks.find_all do |task|
```

Comment:
It seems like you could also use `reject` with `task.completed_at.nil?` as an alternate way to retrieve the same data